### ISSUE TITLE
Feedback on milestone #1

### ISSUE THREAD

**Issue by barryross** [captainspaceman/homework-wireframe#1]
This is fantastic...totally pro and my first impression is that this is a huge improvement over their current one.  Thank you for sending along the Invision link and the Zeppelin docs...not necessary, but amazing!  On that note, I had trouble accessing the Zeppelin link...can you please try inviting me to view it with my `barry@4d-media.ca` email if you would like? 

I would like to help pin down a manageable scope for you to tackle for the final.  I really like your idea of tackling this modal/cta flow as your jQuery implementation.  You may already realize this, but while you can totally build the form, in order for it to be "functional"....for you to receive the submission and send a confirmation...it would need to be wired up to server-side code (you may be able to leverage Mailchimp, but sacrifice the ability to build the form completely from scratch).    Just want you to keep that in mind.  

How many pages are you planning on tackling for the final?  In the Invision project, it looks like there are a number of pages, which is great, but I don't want you to bite off more than you can chew!

Great work on this first milestone and I'm stoked to see this site take shape!

**Comment by captainspaceman** [comment]
Thanks Barry! 

Yeah there's a fair amount of pages but most of them are just content pages for SEO purposes, I've already been a working on an html "template" for them and I think I nailed it last night. So the tricky tasks are going to be styling the homepage to be responsive, sticky nav, and then the jquery challenges ou mentioned above. 

Do you have any experience with Gravity forms? Do they have an API we could leverage for server-side data collection? Something like this?

https://www.gravityhelp.com/documentation/article/gravity-forms-api/

**Comment by barryross** [comment]
You’re welcome, and thanks for the follow-up!  That makes sense regarding the content pages, and that’s great you feel good about your HTML template for them. 

I do have experience with Gravity Forms, but it has all been in the context of Wordpress.  I believe Gravity Forms is exclusively a Wordpress plugin and the API you sent described how to interact with a Gravity Forms installation on Wordpress.  

Ya know, I think this is going to be a solution for you: https://formspree.io/ <https://formspree.io/>

You can point your form to their server, and the info from the form submission will be relayed.

**Comment by barryross** [comment]
Also, thanks for the invite to Zeppelin.  You very well may already be doing this, but I noticed that Zeppelin generates CSS , and I wanted to encourage you to write out the CSS by hand even when using Zeppelin to help generate the values. I think the practice of typing it out is invaluable.

**Comment by captainspaceman** [comment]
Oh rad. 

formspree.io is the way to go. 

Thanks Barry!

Also, yeah I've been noticing shortcomings with Zeplin... the css they generate isn't quite "right" so I've been ignoring it. Oftentimes it's been just flat out wrong. 

Though I have found Zeplin's conversion of pixels to percentages to be REALLY useful. You just click on a section and press "option" - on a mac. It's pretty darn close. 

Thanks for looking this over man. Exciting stuff.

**Comment by barryross** [comment]
Yup, you’re welcome for looking it over!  …and I completely own the fact that I’ve been spelling Zeplin as in "Led Zeppelin”, just realized that one!

That’s great if it helps convert to % for you, that could be very useful!

 It is exciting for sure!
